common/util_test.go

```diff
@@ -12,13 +12,37 @@ import (
 	"time"
 
 	"github.com/stretchr/testify/assert"
+	"gopkg.in/src-d/go-git.v4"
+	"gopkg.in/src-d/go-git.v4/storage/memory"
+	"gopkg.in/src-d/go-git.v4/config"
 )
 
 var (
 	testPrivateKey = []byte("very_sekrit_key")
 	testToken      = "eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.e30.AqFWnFeY9B8jj7-l3z0a9iaZdwIca7xhUF3fuaJjU90"
 )
 
+func getMockRepo(url string) (*git.Repository, error){
+	memory := memory.NewStorage()
+
+	if url[len(url) - len(".git"):] != ".git" {
```

Comment:
Since this is a function purely for testing, I don't think there's really a need to provide `url` as a parameter - I think it should be hardcoded. This would eliminate the need for this URL check and make the behaviour of this test helper more predictable. Otherwise this is pretty clean!

Reply:
Agreed, it would make more sense to have the parameter when the function is reused.

Done.